### PR TITLE
Fix hf-mlm-roberta-b-fine-conv

### PR DESCRIPTION
**Pull request by ManfeiBai**
# Description

fix related to https://github.com/GoogleCloudPlatform/ml-testing-accelerators/pull/917

# Tests

used tests commands from https://github.com/GoogleCloudPlatform/ml-testing-accelerators/pull/917

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: tests/pytorch/nightly/hf-lm.libsonnet [hunk 1]
```diff
@@ -196,6 +196,7 @@ local utils = import 'templates/utils.libsonnet';
       |||,
       tpuVmExtraSetup: |||
         pip install tensorboardX google-cloud-storage
+        pip install --upgrade accelerate
```
Comment (alanwaketan) [inline tests/pytorch/nightly/hf-lm.libsonnet:199]: Do you know if this can be applied to other tests?

Comment (ManfeiBai) [reply tests/pytorch/nightly/hf-lm.libsonnet:199]: not yet tested locally;

if we want to applied to other tests, my stupid solution would be copy to other tpuVmExtraSetup in other tests' tpuVm

Comment (alanwaketan) [reply tests/pytorch/nightly/hf-lm.libsonnet:199]: @will-cromar @JackCaoG Is there an example of how to do this in some common configurations?

Comment (JackCaoG) [reply tests/pytorch/nightly/hf-lm.libsonnet:199]: hmm, maybe https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/pytorch/nightly/common.libsonnet#L97?

**Comment by alanwaketan** [comment]
@ManfeiBai Can you update the PR?

**Comment by ManfeiBai** [comment]
Thanks, @alanwaketan for tracking this ticket, I think https://github.com/GoogleCloudPlatform/ml-testing-accelerators/pull/917/files's modify will be more general with newest setup about `accelerate `

will close this PR and continue track in https://github.com/GoogleCloudPlatform/ml-testing-accelerators/pull/917/

**Pull request closed without merging**